src/main/java/mods/railcraft/common/blocks/machine/epsilon/TileElectricFeeder.java

```diff
@@ -23,7 +25,11 @@
  *
  * @author CovertJaguar <http://www.railcraft.info>
  */
-public class TileElectricFeeder extends TileMachineBase implements IElectricGrid, ISinkDelegate {
+@Optional.InterfaceList(
+        value = { @Optional.Interface(
+                iface = "gregtech.api.interfaces.tileentity.IEnergyConnected",
+                modid = "gregtech"), })
```

Comment:
Unfortunately I believe this has the consequence of breaking compatibility with GT6, I'm looking into if there is a way around this

Reply:
please elaborate why that is the case, you say it changes a signature but i do not see any changed signature of anything i actually use?

Reply:
Tested it and it seems to work fine, disregard